packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -61,7 +61,49 @@ No. You can only set up one custom domain per Okta org.
+
+This method of configuring a custom domain is recommended because Okta will manage your cerificate renewals in imperpetuity, via an integration with Let's Encrypt, a free certificate authority. The certificate procurement process is free, and also faster and easier than configuring a custom domain with your own certificate.
+
+1. In the Admin Console, select **Customizations**, and then **Domain**.
+2. In the **Custom URL Domain** box, click **Edit**.
+3. Click **Get Started** to start the configuration wizard.
+
+### Add your subdomain information
+
+On the Add Domain page of the configuration wizard, enter your subdomain name, for example, `login.example.com`, and then click **Next**. Verifying domain ownership is the next step in the configuration wizard.
+
+### Create a DNS TXT  and CNAME record
+
+You need to add a DNS TXT and CNAME record to your domain to verify ownership of your domain with Okta before Okta can serve traffic over it. This record includes the Okta-generated values provided in the **Host** and **Value** columns of the table on the **Update your DNS page**. Okta verifies that you own your domain when it finds the TXT record that contains the generated value.
+
+1. On the **Update your DNS** page of the configuration wizard, copy the values of the **Host** and **Data** columns into a text file.
+
+2. Sign in to your Domain Name registrar and locate the option to modify your DNS records.
+
+3. Add a TXT record and paste the value that you copied from the **Host** column into the appropriate field, for example, the **Name** or **Host** field.
+
+> **Note**: Depending on your domain provider, you may only need to enter `_oktaverification` rather than `_oktaverification.login.example.com`. If your domain provider doesn't support the value that you enter, verification fails and your custom URL domain configuration is incomplete.
+>
+> You can perform a DNS lookup of your `_oktaverification` DNS record to verify that it's correctly configured. For example, you might use Google's [Dig](https://toolbox.googleapps.com/apps/dig/) tool to check your `_oktaverification.login.example.com` DNS record.
+
+4. Paste the value that you copied from the **Data** column into the appropriate field, for example, the **Record** or **Value** field.
+
+5. Wait for the DNS record to propagate (typically one to five minutes, but it may take longer), and then return to Okta and click **Next** to prove to Okta that you have rights to use the domain name.
+
+> **Note:** It may take up to 24 hours for your DNS changes to propagate. If your changes don't appear within 24 hours, return to this step and confirm your settings. Use a tool like [Dig](https://toolbox.googleapps.com/apps/dig/) to check your DNS records.
+
+6. If **Certificate issued** appears, click **Finish**. If an error occurs, possible issues may be that the TXT record may not have propagated yet or there may be a copy and paste issue with the values. 
```

Comment:
It could be that their CNAME record is incorrect as well or there may be an operational issue with Let's Encrypt which can be checked with https://letsencrypt.status.io/